src/test/java/org/owasp/esapi/reference/EncoderTest.java

```diff
@@ -1050,4 +1051,176 @@ public void testGetDefaultCanonicalizationCodecs() {
 
         assertEquals( true, caughtExpected );   // Verify it threw an IntrusionException
     }
+
+    /**
+     * Test of encodeForJSON method, of class org.owasp.esapi.Encoder.
+     */
+    public void testEncodeForJSON() {
```

Comment:
I've never gotten around to rewriting the tests in here, but in the future I'd like to enforce the idea that we limit 1 test to 1 test method.

Reply:
Done. The monolithic self tests were broken-out into their constituent parts.